tests/datasource/fluent/test_contexts.py

```diff
@@ -69,7 +67,7 @@ def test_add_fluent_datasource_are_persisted(
     assert set_spy.call_count == 1
     cloud_api_fake.assert_call_count(
         f"{GX_CLOUD_MOCK_BASE_URL}/organizations/{FAKE_ORG_ID}/datasources",
-        1,
+        2,
```

Comment:
I think it's because the fixture was overriding context.datasources with a dict, but since removing that override, accessing context.datasources results in a REST call to list datasources.